src/main/java/org/opensearch/security/auth/BackendRegistry.java

```diff
@@ -201,7 +202,11 @@ public boolean authenticate(final SecurityRequestChannel request) {
             .orElse(false);
         if (isBlockedBasedOnAddress) {
             if (isDebugEnabled) {
-                log.debug("Rejecting REST request because of blocked address: {}", request.getRemoteAddress().orElse(null));
+                InetSocketAddress ipAddress = request.getRemoteAddress().orElse(null);
+                log.debug(
+                    "Rejecting REST request because of blocked address: {}",
+                    ipAddress != null ? "/" + ipAddress.getAddress().getHostAddress() : null
```

Comment:
for my understanding can you add a "before" and "after" example of what this change would look like in debug log?

Reply:
I ran into an issue when I was running tests on my mac that assertions like [this](https://github.com/opensearch-project/security/blob/main/src/integrationTest/java/org/opensearch/security/IpBruteForceAttacksPreventionTests.java#L95) were failing because the message in the logs was platform dependent. 

When running on ubuntu, the message in the logs was `Rejecting REST request because of blocked address: 127.0.0.8/127.0.0.8:51549`